src/layout/label.js

```diff
@@ -0,0 +1,46 @@
+import { range } from '../util/scale';
+import { rebindAll } from 'd3fc-rebind';
+import { label as labelLayout } from 'd3fc-label-layout';
+
+export default function(strategy) {
+
+    var adaptee = labelLayout(strategy);
+
+    var xScale = null,
+        yScale = null;
+
+
+    var label = function(selection) {
+        // automatically set the bounds of the strategy based on the scale range
+        if (strategy && strategy.bounds && xScale && yScale) {
+            var xRange = range(xScale),
+                yRange = range(yScale);
+            strategy.bounds([
```

Comment:
Just noticed that bounds only accepts two arguments, I assume that means it's assuming zero as the origin? Maybe that should be changed (separately!)